collector/build.gradle

```diff
@@ -50,6 +58,12 @@ dependencies {
     androidTestImplementation "org.assertj:assertj-core:${rootProject.ext.assertjCoreVersion}"
     testImplementation "org.mockito:mockito-core:${rootProject.ext.mockitoCoreVersion}"
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
+
+
+    testImplementation "org.powermock:powermock-core:2.0.9"
```

Comment:
why do we need powermock? Can't we do this with mockito alone?

Reply:
I needed it for testing private methods, now with refactoring it's not needed any more, so I removed it